integration-tests/state.go

```diff
@@ -283,3 +283,21 @@ func (s System) getValPower(chain uint, validator uint) uint {
 
 	return 0
 }
+
+// TODO: Best solution for default validator fulfilling queries etc. is a dedicated, non validating, full node.
+// See https://github.com/cosmos/interchain-security/issues/263
+func (tr TestRun) getDefaultValidator(chain chainID) validatorID {
+	return alice
```

Comment:
This won't work

Reply:
Stick to the old way, or just implement https://github.com/cosmos/interchain-security/issues/263 in this PR